### ISSUE TITLE
Jspecify mode not allowing @Nullable varargs

### ISSUE THREAD

**Issue by agrieve** [uber/NullAway#1122]
#674 suggests (I think) that this should work, but I'm finding otherwise...

We have:
```java
public class Log {
    public static void d(String tag, String messageTemplate, @Nullable Object @Nullable... args) {
       ...
    }
}
```

And a call:
```hava
Log.d(TAG, "Got message: %s!", message);
```

And a warning:
```
../../content/public/android/java/src/org/chromium/content/browser/sms/SmsVerificationReceiver.java:127: warning: [NullAway] passing @Nullable parameter 'message' where @NonNull is required
                Log.d(TAG, "Got message: %s!", message);
                                               ^
    (see http://t.uber.com/nullaway )
```

**Comment by msridhar** [comment]
I can't reproduce this in 0.12.2 or latest master branch, with or without the JSpecifyMode flag.  Here's my test input:
```java
import org.jspecify.annotations.NullMarked;
import org.jspecify.annotations.Nullable;
@NullMarked
public class VarargsTest {
    public static class Log {
        public static void d(String tag, String messageTemplate, @Nullable Object @Nullable... args) {
        }
    }
    static final String TAG = "AnotherTest";
    public static void callLog(@Nullable Object message) {
        Log.d(TAG, "Got message: %s!", message);
    }
}
```
@agrieve could you let me know what I'm missing?

**Comment by agrieve** [comment]
Confirmed it works fine if `Log` is in the same `javac` invocation. It looks like it fails only when `Log` is provided via `-classpath`.

**Comment by msridhar** [comment]
I'm still having trouble reproducing, even with `Log` in the classpath.  Are you on NullAway 0.12.3 (just released this morning)?  And are you using JSpecify's `@Nullable`?

**Comment by agrieve** [comment]
Using our own TYPE_USE `@Nullable`... But I just tried 0.12.3 and it looks to be fixed! Thanks!

**Comment by msridhar** [comment]
Awesome glad to hear it!
